qa/1906

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+# PCP QA Test No. 1906
+# [what am I here for?]
```

Comment:
Good question!  :)  Maybe something like "Testing pmseries language queries involving scalar operations".
Also, an additional test (with a smaller selection of the cases) run under valgrind might be a good idea here.